src/App.jsx

```diff
@@ -46,7 +49,7 @@ function App() {
             text="Are you sure you want to delete this movie?"
             onModalClose={modal.close}
           >
-            <MyButton className="button__red">confirm</MyButton>
+            <MyButton className="button__red" onClick={() => { dispatch(deleteMovie(modal.data)); modal.close(); }}>confirm</MyButton>
```

Comment:
ideally, we should close modal only when delete operation is done, meaning we show loader and only when request is finished - then hide modal. here we are we sending request and immediately closing the modal